app/javascript/mastodon/components/poll.js

```diff
@@ -99,6 +104,12 @@ class Poll extends ImmutablePureComponent {
     const active             = !!this.state.selected[`${optionIndex}`];
     const showResults        = poll.get('voted') || poll.get('expired');
 
+    let titleEmojified = option.get('title_emojified');
+    if (!titleEmojified) {
+      const emojiMap = makeEmojiMap(poll);
+      titleEmojified = emojify(escapeTextContentForBrowser(option.get('title')), emojiMap);
```

Comment:
This might be slow now. Is it possible to move this to the normalizer?

Reply:
This is for the public pages, where we don't actually use the redux store and normalizer. Could still move it in the public-page specific code, before going through `fromJS` I guess.

Reply:
Oh right. Then it's fine. I completely forgot that's how it was structured.